src/extractor/guidance/turn_handler.cpp

```diff
@@ -705,9 +705,8 @@ void TurnHandler::handleDistinctConflict(const EdgeID via_edge,
         const auto left_classification = node_based_graph.GetEdgeData(left.eid).road_classification;
         const auto right_classification =
             node_based_graph.GetEdgeData(right.eid).road_classification;
-        if (canBeSeenAsFork(left_classification, right_classification))
-            assignFork(via_edge, left, right);
-        else if (left_classification.GetPriority() > right_classification.GetPriority())
+
```

Comment:
Fork does left/right assignment which does not really work in all cases. I wasn't able to reproduce a case for it, but since we are assigning all right/ all left turns here, assigning 'fork left/fork right` doesn't do so well.